octobot_trading/personal_data/orders/channel/orders.py

```diff
@@ -23,10 +23,10 @@
 
 
 class OrdersProducer(exchanges_channel.ExchangeChannelProducer):
-    async def push(self, orders, is_from_bot=False, are_closed=False):
-        await self.perform(orders, is_from_bot=is_from_bot, are_closed=are_closed)
+    async def push(self, orders, is_from_bot=False, are_closed=False, is_from_init=False):
+        await self.perform(orders, is_from_bot=is_from_bot, are_closed=are_closed, is_from_init=is_from_init)
 
-    async def perform(self, orders, is_from_bot=False, are_closed=False):
+    async def perform(self, orders, is_from_bot=False, are_closed=False, is_from_init=False):
```

Comment:
Why not using `is_from_bot` ?

Reply:
There might be order that are not form the bot but that are created after the bot started, in this case we need to update the portfolio available amounts